Training/Program.cs

```diff
@@ -1,7 +1,47 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Chess Board Program
+using System.Text;
+using static System.Console;
+
+OutputEncoding = new UnicodeEncoding ();
+
+// Array containing White and Black coins
+string[] white = new[] { "\u2656", "\u2658", "\u2657", "\u2654", "\u2655", "\u2657", "\u2658", "\u2656", "\u2659" };
+string[] black = new[] { "\u265C", "\u265E", "\u265D", "\u265B", "\u265A", "\u265D", "\u265E", "\u265C", "\u265F" };
+WriteLine ("\u250F" + String.Concat (Enumerable.Repeat ("\u2501\u2501\u2501\u2501\u2501\u2533", 7)) +
+                   "\u2501\u2501\u2501\u2501\u2501\u2513");
+DrawLines ('V');
+PrintCoins (white, 'M');
+WriteLine ();
+DrawLines ('H');
+PrintCoins (white, 'P');
+WriteLine ();
+for (int i = 0; i < 5; i++) {
+   DrawLines ('V');
+   DrawLines ('H');
+   DrawLines ('V');
+}
+PrintCoins (black, 'P');
+WriteLine ();
+DrawLines ('H');
+DrawLines ('V');
+PrintCoins (black, 'M');
+WriteLine ();
+WriteLine ("\u2517" + String.Concat (Enumerable.Repeat ("\u2501\u2501\u2501\u2501\u2501\u253B", 7)) +
+           "\u2501\u2501\u2501\u2501\u2501\u251B");
+
+/// <summary>Method to print the coins</summary>
+/// <param name="piece">Indicates the colour of coins</param>
+/// <param name="type">Main coins (M) or Pawn (P)</param>
+void PrintCoins (string[] pieces, char type) {
+   Write ("\u2503");
+   if (type == 'M')
+      for (int i = 0; i < 8; i++)

```

Comment:
```suggestion
      foreach (var piece in pieces)

```
This will be more readable.

Reply:
Mam I have used for because I have given the Unicode character for pawn in the last of the array. If I use a foreach loop it would also print the pawn. So I am using a for loop

Reply:
I suggested to keep the pawn separately and not part of this array.
Better have two methods PrintCoins and PrintPawns. That way you don't need to access the last element of the array.